app/User.php

```diff
@@ -98,4 +99,25 @@ public function loadouts()
     {
         return $this->hasMany(Loadout::class);
     }
+
+    public static function getUsers(Request $request)
+    {
+        $search = $request->search;
+
+        if ($search == '') {
+            $users = User::orderby('name', 'asc')->select('id', 'name')->limit(5)->get();
+        } else {
+            $users = User::orderby('name', 'asc')->select('id', 'name')->where('name', 'like', '%'.$search.'%')->limit(5)->get();
+        }
+
+        $response = [];
+        foreach ($users as $user) {
+            $response[] = [
+                'id'=>$user->id,
+                'text'=>$user->name,
+            ];
+        }
+
+        return response()->json($response);
```

Comment:
We really shouldn't return the response from here either, that should be done in the controller.

Reply:
This is a remnant of my battle with the search by creator field 😢